### DISCUSSION TITLE
Start rustdesk remote over ssh

### DISCUSSION THREAD

**Discussion by it-da-de** [rustdesk/rustdesk#822] [General]
for the dumbest adoptable user i can connect to ssh as root and want to install and start rustdesk remotely incl. login without any user action. Is this possible? Currently I can connect with ssh and install rustdesk as root. Then I change the user with su - userid. Find the correct display with w and start rustdesk as the user with DISPLAY=:1 rustdesk. Find the id and password in head  ~/.config/rustdesk/RustDesk.toml
I think this should work, but Unfortunately I get X11 error on the local site. Both are on ubuntu 20.04 and wayland is disabled. Any hints?

**Comment by it-da-de** [comment]
Works perfekt, after i restart the service

**Reply by rustdesk** [reply]
Good to know
